### PR TITLE
added tencent support

### PR DESCRIPTION
**Pull request by hanlhan**
added tencent support

### CODE REVIEW

File: deep_translator/tencent.py
Comment (nidhaloff) [inline deep_translator/tencent.py:82]: Why not re-use this from BaseURL in constants.py? or create a variable for it in constants.py. IT would be easier to change it if something changes in the future

File: deep_translator/tencent.py
Comment (nidhaloff) [inline deep_translator/tencent.py:92]: remove or update comment if not relevant anymore

File: tests/test_tencent.py
Comment (nidhaloff) [inline tests/test_tencent.py:67]: `or secret_key is None`?

File: deep_translator/tencent.py
Comment (nidhaloff) [inline deep_translator/tencent.py:134]: remove if not needed anymore

**Review by nidhaloff — CHANGES_REQUESTED** [review]
Thanks for the great PR!
I requested some small updates, other than that everything looks good

**Comment by hanlhan** [comment]
Hi，thanks for your careful and useful review, I have fixed all of this problems, you can review it.

Please let me know if have any other problems.

Thanks for your great work, this repo help me a lot.

**Comment by nidhaloff** [comment]
Thanks for the update @hanlhan . It seems that there are some conflicts after I merged the baidu PR. Everything looks good. I will approve it after you resolve the conflicts

**Pull request merged**
